src/app/models/User.ts

```diff
@@ -93,6 +95,8 @@ export class User extends AbstractModel<IUser> implements IUser {
   public readonly isConfirmed?: boolean;
   public readonly rolesMask?: number;
   public readonly rolesMaskNames?: string[];
+  @bawPersistAttr({ update: true })
```

Comment:
if i remember our DSL, update: true negates create: true.

It should be writeable on creation, though it will probably cause an error. 

otherwise leave a note and a todo: when api gets proper users api, then we can fix this

Reply:
![image](https://github.com/user-attachments/assets/18b22526-8c09-47a4-b930-2c1d8d6f8956)